example/c10_custom_cs_example.cpp

```diff
@@ -79,24 +63,26 @@ int main()
     mars_point viking1(-48.23, 22.54); // Viking 1 landing site in Chryse Planitia
     mars_point pathfinder(-33.55, 19.33); // Pathfinder landing site in Ares Vallis
 
-    double d = boost::geometry::distance(viking1, pathfinder); // Distance in radians on unit-sphere
-
-    // Using the Mars mean radius
+    // To calculate distance, declare and construct a strategy with Mars mean radius, in KM
     // (Source: http://nssdc.gsfc.nasa.gov/planetary/factsheet/marsfact.html)
+    boost::geometry::strategy::distance::haversine<double> const haversine(3389.5);
```

Comment:
We have umbrella strategies now. Shouldn't we use them in the example? In this case `strategies::distance::spherical<>` ?

Reply:
OK I'll try.
The "problem" was that these original examples didn't work anymore. So somewhere we were not backwards compatible, the strategy converters did not work for these cases.
Anyway, I was a bit short on time and fixed it with the least amount of time at that moment. I'll look at it again, but anyway, you're welcome to update them again if you think it's better or better illustrates the possibilities.

Reply:
Yes, we should probably find out why this is not backward compatible.

Reply:
> We have umbrella strategies now. Shouldn't we use them in the example? In this case `strategies::distance::spherical<>` ?

That works